content/docs/cucumber/varieties.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Cucumber Varieties
+subtitle: "Varieties and Semantic versioning"
+
+weight: 3
+---
+
+Every [Cucumber implementation](/docs/installation) has its own codebase, release cycle and [semver](https://semver.org/) version that is independent from other Cucumber implementations.
+
+The Cucumber team also maintains a list of *varieties* which defines essential characteristics of various Cucumber versions. A Cucumber release may satisfy zero or more characteristics of a variety. If it satisfies all of them it is a member of that variety.
+
+Marking Cucumber releases as part of a variety serves several purposes:
+
+* Define stretch goals for upcoming Cucumber implementations
+* Allow tool vendors to support particular varieties with more ease
+* Documentation becomes easier to read and write as it applies to all releases in the variety
+
+Varieties are named after Wikipedia's [list of Cucumber varieties](https://en.wikipedia.org/wiki/List_of_cucumber_varieties). Higher alphabetical order means more recent.
+
+# Babylon
+
+Characteristics               | [Cucumber-JVM 5.0.0-RC2] | [Cucumber.js master] | [Cucumber-Ruby 4.0.0.rc.2]
```

Comment:
Cucumber JVM 5.0.0-RC2 hasn't been released yet. We should only include released versions here. Otherwise this becomes a speculative document rather then a reference.